lib/api/websiteGet.js

```diff
@@ -44,6 +46,12 @@ function _errorActions(err, errorDocument, routingRules,
                     // to true
                     return callback(err, true, null, corsHeaders);
                 }
+                // do not return the error object if it is private
```

Comment:
is the idea that you just return the default error message if the object is private rather than sending a stored error file?  please clarify